virtual_rainforest/models/litter/litter_model.py

```diff
@@ -101,6 +124,21 @@ def __init__(
             LOGGER.error(to_raise)
             raise to_raise
 
+        # Check that lignin proportions are between 0 and 1
+        if (
+            np.any(data["lignin_above_structural"] < 0.0)
+            or np.any(data["lignin_woody"] < 0.0)
+            or np.any(data["lignin_below_structural"] < 0.0)
+            or np.any(data["lignin_above_structural"] > 1.0)
+            or np.any(data["lignin_woody"] > 1.0)
+            or np.any(data["lignin_below_structural"] > 1.0)
+        ):
```

Comment:
For this (an the previous check on the pools), maybe make it more informative such that the user knows which specific array(s) have wrong values. Something like:

```python
wrong = []
for var in vars_of_interest:
    if var < 0 or var > 1:
        wrong.append(var)

if wrong:
    to_raise = InitialisationError(
                f"Proportions in {wrong} must be between 0 and 1!"
            )
    LOGGER.error(to_raise)
    raise to_raise
```

Or something along those lines.